### ISSUE TITLE
DLL export macro

### ISSUE THREAD

**Issue by jlaumon** [ocornut/str#5]
Hey,

Are you interested in getting a pull request that adds a macro for wrapping the DLL export/import keywords?

eg.
```c++
#ifndef IMSTR_API
#define IMSTR_API
#endif

...

class IMSTR_API Str
...
```

**Comment by ocornut** [comment]
Yes! Note however that the prefix we use is STR (not IMSTR) here.
Thank you!

**Comment by jlaumon** [comment]
Noted! Thx

**Comment by ocornut** [comment]
Closing (fixed by #7)
